pytorch_lightning/loggers/logger.py

```diff
@@ -346,6 +346,16 @@ def __iter__(self):
         # if DummyLogger is substituting a logger collection, pretend it is empty
         yield from ()
 
+    def __getattr__(self, name):
+        """Allows the DummyLogger to be called with arbitrary methods, to avoid AttributeErrors."""
+
+        def method(*args, **kwargs):
+            rank_zero_info(
+                f"The DummyLogger is being used and the method `{name}` was called on it but it does not exist"
```

Comment:
Let's mention fast_dev_run
```suggestion
                f"The `DummyLogger` is being used and the method `{name}` was called on it but it does not exist."
                " This is the case when `fast_dev_run` is enabled"
```